pkg/operator/ceph/client/controller.go

```diff
@@ -117,16 +117,19 @@ func createClient(context *clusterd.Context, p *cephv1.CephClient) error {
 		return errors.Wrapf(err, "failed to generate client entity %q", p.Name)
 	}
 
+	// assume we have been given the admin key for configuring cephx clients
+	clusterInfo := cephclient.AdminClusterInfo(p.Namespace)
```

Comment:
Fail if not the case? This might be more obvious for the user.

Reply:
Actually I meant to update this to call LoadClusterInfo() instead of assuming the admin user. fixing it...